packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -1930,6 +1945,13 @@ class CupertinoTimerPicker extends StatefulWidget {
   /// Defaults to null, which disables background painting entirely.
   final Color? backgroundColor;
 
+  /// The uniform height of all children.
+  ///
+  /// All children will be given the [BoxConstraints] to match this exact
+  /// height. Must not be null and must be positive.
+  /// Default to a value that matches the default iOS date picker wheel.
```

Comment:
This looks like it's exactly the same as the other itemExtent docs above, can you use a `@template` and `@macro` to reuse it?